### DISCUSSION TITLE
Replaced CC2531 by a CC2652RB

### DISCUSSION THREAD

**Discussion by goedef** [Koenkk/zigbee2mqtt#5889] [Devices]
I replaced my CC2531 Coordinator with a CC2652RB from Tindie. The processor is much faster and it can support much more devices. After some issues and info from the website I was able to get the new coordinator working. Great!

I had to change the pan_id because Zigbee2mqtt would not start without it. 

Because of the limited signal of the CC2531 I build (a year ago) the CC2530 + CC2591 router from the website and it worked great. Now with the new coordinator the CC2530 router is not visible anymore in the network topology. I assume it has to do with the changed pan_id. Is it possible to get the CC2530 router back in the network again.

Advise would really be appreciated.

**Comment by Koenkk** [comment]
Switching from a CC2531 to CC2652RB requires repairing all devices (https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices). Repairing your CC2530 router should fix it: https://github.com/Koenkk/Z-Stack-firmware/tree/master/router#pairing

**Comment by goedef** [comment]
Koen,

 

Thank you so much! Works like a charm. 

My compliments for the great work you are doing on Zigbee2mqtt.

 

Fred

 

Van: Koen Kanters <notifications@github.com>
Beantwoorden - Aan: Koenkk/zigbee2mqtt <reply@reply.github.com>
Datum: zondag 24 januari 2021 om 19:34
Aan: Koenkk/zigbee2mqtt <zigbee2mqtt@noreply.github.com>
CC: Fred Goede <fred@degoede.nu>, Author <author@noreply.github.com>
Onderwerp: Re: [Koenkk/zigbee2mqtt] Replaced CC2531 by a CC2652RB (#5889)

 

Switching from a CC2531 to CC2652RB requires repairing all devices (https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices). Repairing your CC2530 router should fix it: https://github.com/Koenkk/Z-Stack-firmware/tree/master/router#pairing

**Comment by m0wlheld** [comment]
@goedef How did you proceed when re-pairing your devices, was it required to remove them from their installation location? How many devices needed to be re-paired?

**Comment by goedef** [comment]
Christoph,

 

I had to repair all my devices (~20). It was not necessary to remove them.

 

Fred

 

 

Van: Christoph Dahlen <notifications@github.com>
Beantwoorden - Aan: Koenkk/zigbee2mqtt <reply@reply.github.com>
Datum: maandag 25 januari 2021 om 09:53
Aan: Koenkk/zigbee2mqtt <zigbee2mqtt@noreply.github.com>
CC: Fred Goede <fred@degoede.nu>, Mention <mention@noreply.github.com>
Onderwerp: Re: [Koenkk/zigbee2mqtt] Replaced CC2531 by a CC2652RB (#5889)

 

@goedef How did you proceed when re-pairing your devices, was it required to remove them from their installation location? How many devices needed to be re-paired?

**Comment by m0wlheld** [comment]
Thanks. But you had to unpair all of them beforehand, did you? I can't imagine how they will become willing to pair otherwise, except factory resetting.

**Reply by DrTron** [reply]
Isn't that how you repair most devices anyhow? Hold the button until the LED flashes which is a factory reset?
At least that's how I repaired mine when I changed coordinators.

**Reply by m0wlheld** [reply]
Factory reset is only required (to my understanding) if the device is still paired with another network. If I [remove](https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html#zigbee2mqttbridgerequestdeviceremove) that device previously, it will be open for pairing without factory reset. 

"Holding the button until the LED flashes" is true for most battery-powered devices, but the OSRAM/LEDvance, IKEA Trådfri and Innr light bulbs require a sequence of power on/power off cycles. Hard to achieve (without mounting), if they are installed in lanterns along the alley.

**Reply by DrTron** [reply]
Indeed. In that case I'd unpair them with a mqtt-publish to the coordinator. After that, they should willingly join the new network. For battery devices, that may take some time, but mains-powered should join rather quickly, I think.

**Reply by m0wlheld** [reply]
I hope so, yet it will be difficult as the outermost devices are way out of reach for the coordinator. 

I‘ll have to remove devices from far to narrow, the start over from the inside to outside.

**Comment by goedef** [comment]
Christoph,

 

Maybe I should have, but I did not. I re-paired all devices one by one and it worked ok. 

 

Fred

 

Van: Christoph Dahlen <notifications@github.com>
Beantwoorden - Aan: Koenkk/zigbee2mqtt <reply@reply.github.com>
Datum: maandag 25 januari 2021 om 11:42
Aan: Koenkk/zigbee2mqtt <zigbee2mqtt@noreply.github.com>
CC: Fred Goede <fred@degoede.nu>, Mention <mention@noreply.github.com>
Onderwerp: Re: [Koenkk/zigbee2mqtt] Replaced CC2531 by a CC2652RB (#5889)

 

Thanks. But you had to unpair all of them beforehand, did you? I can't imagine how they will become willing to pair otherwise, except factory resetting.
